bank/bank-backend/src/runtime.test.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import axios, { AxiosError } from "axios";
+import { spawn, execSync, ChildProcess } from "child_process";
+import { Writable } from "stream";
+
+async function waitForMessageTest(command: ChildProcess, port: string) {
+  const stdout = command.stdout as unknown as Writable;
+  const stdin = command.stdin as unknown as Writable;
+  const stderr = command.stderr as unknown as Writable;
+
+  const waitForMessage = new Promise<void>((resolve, reject) => {
+    const onData = (data: Buffer) => {
+      const message = data.toString();
+      process.stdout.write(message);
+      if (message.includes("Server is running at")) {
+        stdout.off("data", onData); // remove listener
+        resolve();
+      }
+    };
+
+    stdout.on("data", onData);
+    stderr.on("data", onData);
+
+    command.on("error", (error) => {
+      reject(error); // Reject promise on command error
+    });
+  });
+  try {
+    await waitForMessage;
+    const response = await axios.get(`http://127.0.0.1:${port}/api/greeting/`).catch((reason) => { return (reason as AxiosError).response!.status; });
+    expect(response).toBe(401);  // Should return with 401: protected resources.
+  } finally {
+    stdin.end();
+    stdout.destroy();
+    stderr.destroy();
+    command.kill();
+  }
+}
+
+describe("runtime-tests", () => {
+  beforeAll(() => {
+    const bankSchema = process.env.BANK_SCHEMA || 'public';
+    if (!bankSchema) {
+      throw new Error("Env 'BANK_SCHEMA' not set!");
```

Comment:
I assume this will never happen because we set the bank schema to public by default. I think we might want to get rid of `BANK_SCHEMA` completely, because it's confusing and not a feature we support.